src/cdk/testing/test-element.ts

```diff
@@ -101,8 +101,12 @@ export interface TestElement {
    */
   sendKeys(modifiers: ModifierKeys, ...keys: (string | TestKey)[]): Promise<void>;
 
-  /** Gets the text from the element. */
-  text(): Promise<string>;
+  /**
+   * Gets the text from the element.
+   * @param options Options that affect what text is included
+   *   exclude: A selector for elements whose text should be excluded from the result.
```

Comment:
It's called `exclude` in the comment but `excludes` in the code (personally prefer `exclude`)